src/charm.py

```diff
@@ -355,7 +356,7 @@ def update_quorum(self, event: EventBase) -> None:
 
             if self.cluster.all_units_quorum:
                 logger.debug("all units running desired encryption - removing upgrading")
-                self.app_peer_data.update({"upgrading": ""})
+                self.app_peer_data.update({"tls-upgrading": ""})
```

Comment:
VERY good catch! Exactly what I was expecting would happen with the upgrade lib, thanks you so much for picking up on that.

Agreed on having slightly more manageable peer-relation management, though in this case, whether it's some other place or here, the 'naming' of 'upgrading' is the issue, so it would have to be multi-line anyway.

Regardless, will remove from this PR.